.github/workflows/ci.yml

```
@@ -7,11 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
-        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_7.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x, 3.0.x]
```

the CI is failing because it couldn't find version `2.4.x`, maybe we need to change it to

```suggestion
        ruby_version: [2.4, 2.5, 2.6, 2.7, 3.0]
```
as it it's here https://github.com/rootstrap/yaaf/blob/master/.github/workflows/ci.yml#L12